Wireframe/index.html

```diff
@@ -10,24 +10,92 @@
+          <br>
+          Wireframes can range from low-fidelity (simple sketches) to high-fidelity (detailed mockups), but they all focus on the layout and functionality rather than the visual design.
+          <br>
+          <br>
+          In simple terms, wireframes are visual representations of a web page or app interface, stripped down to its bare bones. Think of it as the architectural blueprint of your design project. They outline the structure and functionality of your product without getting caught up in colors, visuals, or specific content.
         </p>
-        <a href="">Read more</a>
+        <a href="https://careerfoundry.com/en/blog/ux-design/what-is-a-wireframe-guide/#:~:text=A%20wireframe%20is%20a%20two,are%20kept%20to%20a%20minimum." target="_blank">Read more</a>
       </article>
     </main>
+    <section class="side-images">
+      <div class="bottom-side">
+        <article>
+          <img src="placeholder.svg" alt="" />
+          <h2>Branch in Git</h2>
+          <p>In Git, a branch is a new/separate version of the main repository. A branch in Git is like a separate version of your project where you can work on changes      without affecting the main codebase. It allows you to experiment, develop new features, or fix bugs independently, and later you can merge those changes back into the main project.
+            <br>
+            <br>
+            You always create a branch from an existing branch. Typically, you might create a new branch from the default branch of your repository. You can then work on this new branch in isolation from changes that other people are making to the repository. A branch you create to build a feature is commonly referred to as a feature branch or topic branch. For more information, see Creating and deleting branches within your repository.
+            <br>
+            <br>
+            You can also use a branch to publish a GitHub Pages site. For more information, see About GitHub Pages.
+            <br>
+            You must have write access to a repository to create a branch, open a pull request, or delete and restore branches in a pull request. For more information, see Access permissions on GitHub.
+            <br>
+            <br>
+            In summary, branches in Git allow developers to work on different tasks in isolation, organize development, and merge changes in an efficient and controlled way. They help manage multiple versions of a project simultaneously, making it easier to collaborate and experiment.
+            <br>
+            <br>
+          </p>
+             
+          <a href="https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches" target="_blank">Read more
+
+          </a>
+        </article>
+      </div>
+
+      <div class="bottom-side">
+        <article>
+          <img src="placeholder.svg" alt="" />
```

Comment:
Can you tell me  why we have to use alt tags in img elements, please do  not forget to fill alt tags in images